### PR TITLE
Fix for error #122

### PR DESCRIPTION
**Pull request by MrDunne**
It's not a particularly attractive fix but a fine one all the same. Also cleared 
up some of the imports which were necessary/unnecessary.

### CODE REVIEW

**Comment by MrDunne** [comment]
The last 'fix' was quite ugly. This one is a little nicer. Also more dependency clean-ups

**Comment by MarkVink** [comment]
Personally, I wouldn't import multiple packages entirely using *. It can break when a class with the same name is added to those packages. Why not be specific in what we need to import? I just let my IDE take care of the imports and don't bother with it. But hey, just my opinion :)

**Comment by MrDunne** [comment]
@svenkubiak I'm happy for you to make the call here, I'll update 
the dependency format if you feel the same as @MarkVink 

Personally I don't think it's much of an issue as most IDEs are 
pretty good at dependency resolving if such an error did occur 
so it wouldn't take more than a moment to fix it, so I'd personally 
opt for this as it looks cleaner to me.

File: mangooio-core/src/main/java/io/mangoo/configuration/Config.java [hunk 1]
```diff
@@ -554,7 +552,11 @@ public int getExecutionPool() {
      * @return application.controller from application.yaml or default value if undefined
      */
     public String getControllerPackage() {
```
Comment (svenkubiak) [inline mangooio-core/src/main/java/io/mangoo/configuration/Config.java]: I am not sure if the Config class is the right place for this. IMHO, the config should return the raw values. This should be a new method in the Bootstrap class.

**Comment by svenkubiak** [comment]
I think it did provide two or three PRs for Ninja with cleanly organized imports without wildcard imports :-D Besides, wildcard imports are a sonar issue.

File: mangooio-core/src/main/java/io/mangoo/configuration/Config.java (hunk 1 above)
Comment (MrDunne) [inline mangooio-core/src/main/java/io/mangoo/configuration/Config.java]: In this case I can move it to the bootstrap class. I'll get that done later today

**Comment by MrDunne** [comment]
Sorry I'm not quite sure what your answer is

**Comment by svenkubiak** [comment]
> Sorry I'm not quite sure what your answer is

Please don't use wildcard imports ;-)

**Comment by MrDunne** [comment]
Ok, that will be fixed at the same time later then. I'll bear it in mind in future.

**Comment by MrDunne** [comment]
Assuming Travis verifies the code runs (doing this on the github website), this now fits the guidelines.

**Comment by MrDunne** [comment]
There you go (Y)

**Comment by svenkubiak** [comment]
Thanks!

**Pull request merged**